Envfile

```diff
@@ -42,20 +42,24 @@ variable :ELASTICSEARCH_URL, :String, default: "http://localhost:9200"
 # SSL config
 variable :FORCE_SSL_IN_RAILS, :String, default: "not applicable in dev"
 
-################################################
-############## 3rd Party Services ##############
-################################################
-
-# It's best if you have the following configured
-# for more development experience
-
-# GitHub
+###############################################################
+############## 3rd Party Authentication Services ##############
+###############################################################
+
+# Apple for authentication
+# https://developer.apple.com/sign-in-with-apple/
+variable :APPLE_CLIENT_ID, :String, default: ""
```

Comment:
Need to move these to the new `.env_sample` file

Reply:
We actually need to completely remove this file and transfer these to `.env_sample` now